packages/cojson/src/coValueCore/coValueCore.ts

```diff
@@ -213,6 +219,16 @@ export class CoValueCore {
     }
   }
 
+  unmount() {
+    this.counter.add(-1, { state: this.loadingState });
+
+    if (this.groupInvalidationSubscription) {
+      this.groupInvalidationSubscription();
+      this.groupInvalidationSubscription = undefined;
+    }
+    this.listeners.clear();
```

Comment:
Isn't it contradictory that we're only calling unmount if listeners is empty, but also clear it here?